compiler/codegen/codegen.cpp

```diff
@@ -2390,17 +2391,26 @@ static void codegenPartOne() {
 
 // Do this for GPU and then do for CPU
 static void codegenPartTwo() {
-  if( fLlvmCodegen ) {
+
+  // Initialize the global gGenInfo for C code generation.
+  gGenInfo = new GenInfo();
+
+  // This function assumes the existence of a "_main.c" file before it
+  // actually exists (it is populated down below). In '--llvm' compiles,
+  // the source files this function produces are compiled by a call to
+  // 'runClang(NULL)', and the "_main.c" file is not included in the
+  // client or server bundles.
```

Comment:
I'm finding this comment confusing. The code in mli.cpp seems to have conditionals for LLVM compiles to not use _main.c yet.

Reply:
Removed.